src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceParameterCodeRefactoringProvider.cs

```diff
@@ -39,6 +39,11 @@ protected override bool IsDestructor(IMethodSymbol methodSymbol)
             return false;
         }
 
+        protected override SyntaxNode GetMethodSyntax(SyntaxNode node)
+        {
+            return node.Parent!;
```

Comment:
this is pretty wonky.  there's nothing about a node's parent that means it's a method.